service/src/test/java/com/ge/predix/acs/policy/evaluation/cache/AbstractPolicyEvaluationCacheTest.java

```diff
@@ -102,8 +102,13 @@ public void testGetWithCacheMissForResource() {
         assertNull(this.cache.get(key));
     }
 
-    @Test
-    public void testGetWithCacheHit() {
+    @DataProvider
+    private Object[][] effects() {
```

Comment:
@anubhavi25, wouldn't it better to have a provider of not just effects but the whole policy evaluation results:

return new Object[][] { new Object[] { mockPermitResult() }, { mockIndeterminateResult() } };

This way we will avoid the if statement in line 124-125 below.

Reply:
Agreed fixed